src/languageSupport/languages/flux/lsp/monaco.flux.lsp.ts

```diff
@@ -71,8 +72,10 @@ export function initLspWorker() {
     return
   }
   if (window.Worker) {
+    event('Used LSP worker.')
```

Comment:
I don't see this function in scope. Where does this go? How can we check up on it?

Additionally, I'm not sure we care when we use the native worker. That's the happy path. I just want to know if we have users that are out there using browsers from 2011.

Reply:
Yes, it's in scope. see:
```
import {event} from 'src/cloud/utils/reporting'
```

It gets reported with our amplitude metrics.

Reply:
Reporting both numbers, web worker versus fallback worker, so we can examine as a ratio. Then we can easily justify with "X ratio of users are...".

Reply:
Oh shoot, I must've just scanned over it in this patch. I saw it, wondered what it did, so I went to my vscode (where I already had this file open) and tried to walk it back. Sorry about that!

Reply:
> Reporting both numbers, web worker versus fallback worker, so we can examine as a ratio.

I'm not sure we care about a ratio. I'm mostly interested in "does this code get used _at all_." I have a suspicion that it probably doesn't, but if we need to figure out how to move on from it, that's a completely different task. If we look at this in two weeks and see no traffic, we don't even have to care about the happy path.

I'm happy to leave it in, but I just want to be clear that it is a little more than we need right now.

Reply:
Yeah. I'm just hoping that if it's use at all...the ratio can be so tiny that we decide not to support. 🤷🏼  Just a thought.